OpenRA.Game/Map/MapPreview.cs

```diff
@@ -226,8 +226,9 @@ public void UpdateFromMap(IReadOnlyPackage p, IReadOnlyPackage parent, MapClassi
 					return Ruleset.Load(modData, this, TileSet, ruleDefinitions, weaponDefinitions,
 						voiceDefinitions, notificationDefinitions, musicDefinitions, sequenceDefinitions);
 				}
-				catch
+				catch (Exception e)
 				{
+					Log.Write("debug", "Failed to load rules for {0} with error {1}", Title, e.Message);
```

Comment:
Please either add a colon before the `{1}` or put it in quotes. Quotes also for the title, please. I'd add `package.Name` somewhere in there as well, probably before or after the title.